docs/source/usage/workflow-installation.rst

```diff
@@ -73,7 +73,8 @@ the command ``cylc gui``.
 
     # Run workflow
     cylc vip .
-    # View workflow progress
+    # View workflow progress and don't use & at the end to enable
```

Comment:
Regarding the actual content of this message, it may be better to advices to run cylc gui in a separate terminal, so it can be minimised and reused between workflow runs, rather than having to killed and restarted between commands.

Reply:
Probably want a reminder about `export CYLC_VERSION=8` for that though.